tests/bluetooth/classic/sdp_c/testcase.yaml

```diff
@@ -0,0 +1,16 @@
+tests:
+  sdp.c:
+    platform_allow:
+      - native_sim
+      - native_sim/native/64
+    integration_platforms:
+      - native_sim
+    tags:
+      - bluetooth
+      - sdp
+    harness: pytest
+    harness_config:
+      pytest_dut_scope: session
+      fixture: bumble
```

Comment:
what are you trying to do here? from description it seems that soft has to be running on the host before the test is executed. Fixture in yaml refers to additional hw requirements (e.g. a board have gpio loopback done) to select which board is capable of running the test https://docs.zephyrproject.org/latest/develop/test/twister.html#fixtures. In your case I think you can write a "bumble" pytest-fixture which will start a required subprocess and close it after the test is done.

Reply:
the test can detect if the "bumble" is available on host and if not mark the test as skipped due to lack of it.

Reply:
maybe this will help you getting some ideas https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/net/lib/lwm2m/interop/pytest/conftest.py#L56

Reply:
I run the case with the command line `west twister -v -p mimxrt1170_evk@B/mimxrt1176/cm7 --device-serial COM4 -T tests/classic/sdp_c --west-flash --west-runner=jlink -X bumble:usb:0`. 

As I know, the arg `-X bumble:usb:0` is used to pass the argument `usb:0` to fixture. 
Please see it here, https://github.com/zephyrproject-rtos/zephyr/pull/82316/commits/3ef6b2956dc18dcfe05d159fac5b26eee80400ec#diff-c0436aa098e922696a5603141d1bfa524eb92b1c43f8fe3d8d65e44ce06cbf6aR26-R29